### ISSUE TITLE
plugin woocomerce problem

### ISSUE THREAD

**Issue by NLSARCOS** [2Checkout/woocommerce#17]
Hello, I have this plugins in woocomerce and I like the convert plus method but for a couple of days when my clients try to make a payment the plugins instead of sending it to the 2checkout page it sends it to my wordpress administrator page.

The inline plugins always give me an error and the person has to go through the purchase process twice and usually that causes disagreement.

I would appreciate a help.

**Comment by matheus-morganti** [comment]
Same here!!

**Comment by craigchristenson** [comment]
Thanks for reporting the issues. I am unable to replicate anything like this, it could be 2Checkout account related. Can you provide a screen shot of the error you are getting?

**Comment by NLSARCOS** [comment]
![image](https://user-images.githubusercontent.com/80655069/112027976-b6a07b00-8b05-11eb-83f9-f29dab714273.png)

This error shows me within the 2checkout platform.

When I activate the convertPlus plugins it does not redirect me to the payment page, but it sends me to the wordpress administrator 

I would appreciate your prompt help since I need the convertplus, since I sell to several countries and people feel more comfortable making the payment with their local currency.

**Comment by NLSARCOS** [comment]
Here is a video of the error when I try to enable ConvertPlus

https://www.loom.com/share/c4d640aa5e2e43498c14fbecdbbf28ec

**Comment by matheus-morganti** [comment]
Yes, that's exactly the same issue I'm facing! I now tried ConvertPlus on another website and is showing the same error.
